dipy/tracking/local/localtracking.py

```diff
@@ -93,12 +100,22 @@ def _generate_streamlines(self):
             directions = dg.initial_direction(s)
             directions = directions[:max_cross]
             for first_step in directions:
-                stepsF = local_tracker(dg, tc, s, first_step, vs, F, ss, fixed)
-                if stepsF < 0:
+                stepsF, tissue_class = local_tracker(dg, tc, s, first_step,
+                                                     vs, F, ss, fixed)
+                # enum TissueClass (tissue_classifier.pyd) is not accessible
```

Comment:
yes. I added a comment in the code where we use hard-coded tissue types. I will make the issue once merged.